tests/serverpod_test_client/lib/src/protocol/client.dart

```diff
@@ -1440,15 +1440,15 @@ class EndpointMapParameters extends _i1.EndpointRef {
       caller.callServerEndpoint<Map<int, int>>(
         'mapParameters',
         'returnIntIntMap',
-        {'map': map},
+        {'map': map.isEmpty ? [] : map},
```

Comment:
This is the first half of the fix (for sending such maps).

Interestingly, there was only 1 endpoint making use of such parameters (and until now no empty maps were provided as arguments).